File: php/006/main.php
<?php
/*
 * ProjectEuler 006
 * PHP / main.php
 * copyright (c) 2015 Susisu
 *
 * PHP 5.4.30
 * `php main.php`
 */

function main() {
    $s = 0;
    $t = 0;
    for ($n = 1; $n <= 100; $n++) {
        $s += $n;
        $t += $n * $n;
    }
    echo $s * $s - $t;
    echo "\n";
}

main();

File: ruby/006/main.rb
=begin
    ProjectEuler 006
    Ruby / main.rb
    copyright (c) 2015 Susisu

    Ruby 2.0.0p481
    `ruby main.rb`
=end

def main
    s = 0
    t = 0
    for i in 1 .. 100
        s += i
        t += i * i
    end
    puts s * s - t
end

main()
